services-custom/s3-transfer-manager/src/main/java/software/amazon/awssdk/transfer/s3/model/ResumableFileDownload.java

```diff
@@ -61,6 +63,7 @@ public final class ResumableFileDownload implements ResumableTransfer,
     private final Instant s3ObjectLastModified;
     private final Long totalSizeInBytes;
     private final Instant fileLastModified;
+    private final List<Integer> completedParts;
```

Comment:
Is the reason for List that we may download different parts in parallel in the future?

Reply:
yes, trying to prepare for when we will do parallel write to file